plugin-niassa+pinery/src/main/java/ca/on/oicr/gsi/shesmu/pinery/PinerySource.java

```diff
@@ -154,6 +155,9 @@ private ItemCache(Path fileName) {
                     maybeGetRunField(run, RunDto::getSequencingKit),
                     maybeGetRunField(run, RunDto::getContainerModel),
                     Optional.empty(),
+                    run.getStartDate() == null || run.getStartDate().isEmpty()
+                        ? Instant.EPOCH
+                        : ZonedDateTime.parse(run.getCreatedDate()).toInstant(),
```

Comment:
Looks like this is supposed to be `run.getStartDate()` rather than `run.getCreatedDate()`? Same below on L233

Reply:
Indeed. :dizzy_face: